src/Symfony/Component/Form/Resources/translations/validators.sq.xlf

```diff
@@ -0,0 +1,139 @@
+                <source>This value is not a valid HTML5 color.</source>
+                <target>Kjo vlerë nuk është një ngjyrë e vlefshme HTML5.</target>
+            </trans-unit>
+            <trans-unit id="100">
+                <source>Please enter a valid birthdate.</source>
+                <target>Ju lutemi shkruani një datëlindje të vlefshme.</target>
+            </trans-unit>
+            <trans-unit id="101">
+                <source>The selected choice is invalid.</source>
+                <target>Opsioni i zgjedhur është i pavlefshëm.</target>
+            </trans-unit>
+            <trans-unit id="102">
+                <source>The collection is invalid.</source>
+                <target>Koleksioni është i pavlefshëm.</target>
+            </trans-unit>
+            <trans-unit id="103">
+                <source>Please select a valid color.</source>
+                <target>Ju lutemi zgjidhni një ngjyrë të vlefshme.</target>
+            </trans-unit>
+            <trans-unit id="104">
+                <source>Please select a valid country.</source>
+                <target>Ju lutemi zgjidhni një shtet të vlefshëm.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>Please select a valid currency.</source>
+                <target>Ju lutemi zgjidhni një monedhë të vlefshme.</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>Please choose a valid date interval.</source>
+                <target>Ju lutemi zgjidhni një interval të vlefshëm të datës.</target>
+            </trans-unit>
+            <trans-unit id="107">
+                <source>Please enter a valid date and time.</source>
+                <target>Ju lutemi shkruani një datë dhe orë të vlefshme.</target>
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Please enter a valid date.</source>
+                <target>Ju lutemi shkruani një datë të vlefshme.</target>
+            </trans-unit>
+            <trans-unit id="109">
+                <source>Please select a valid file.</source>
+                <target>Ju lutemi zgjidhni një skedar të vlefshëm.</target>
+            </trans-unit>
+            <trans-unit id="110">
+                <source>The hidden field is invalid.</source>
+                <target>Fusha e fshehur është e pavlefshme.</target>
+            </trans-unit>
+            <trans-unit id="111">
+                <source>Please enter an integer.</source>
+                <target>Ju lutemi shkruani një numër të plotë.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Please select a valid language.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Ju lutemi zgjidhni një lokale të vlefshme.</target>
```

Comment:
Can you double check if this is correct?

Reply:
Yes, this is correct. Just as an explanation, from Google Translate the word `lokal` means a place where you can have a coffee and the correct word is `lokale` in this context

Reply:
Okey. Cool!